diff --git a/components/InngestStepTools.d.ts b/components/InngestStepTools.d.ts
index 62ddf59d1cd986b71c4a05d60b8c68d52328fe5a..fbe7bff28d0de22f48100f2d2d0f169cc6b3b2ca 100644
--- a/components/InngestStepTools.d.ts
+++ b/components/InngestStepTools.d.ts
@@ -139,7 +139,7 @@ export declare const createStepTools: <TClient extends Inngest.Any>(client: TCli
      * TODO Middleware can affect this. If run input middleware has returned
      * new step data, do not Jsonify.
      */
-    SimplifyDeep<Jsonify<TFn extends (...args: Parameters<TFn>) => Promise<infer U> ? Awaited<U extends void ? null : U> : ReturnType<TFn> extends void ? null : ReturnType<TFn>>>>;
+    SimplifyDeep<TFn extends (...args: Parameters<TFn>) => Promise<infer U> ? Awaited<U extends void ? null : U> : ReturnType<TFn> extends void ? null : ReturnType<TFn>>>;
     /**
      * AI tooling for running AI models and other AI-related tasks.
      */
@@ -164,7 +164,7 @@ export declare const createStepTools: <TClient extends Inngest.Any>(client: TCli
          * TODO Middleware can affect this. If run input middleware has returned
          * new step data, do not Jsonify.
          */
-        SimplifyDeep<Jsonify<TFn extends (...args: Parameters<TFn>) => Promise<infer U> ? Awaited<U extends void ? null : U> : ReturnType<TFn> extends void ? null : ReturnType<TFn>>>>;
+        SimplifyDeep<TFn extends (...args: Parameters<TFn>) => Promise<infer U> ? Awaited<U extends void ? null : U> : ReturnType<TFn> extends void ? null : ReturnType<TFn>>>;
         /**
          * Models for AI inference and other AI-related tasks.
          */
